src/components/Input/Dropdown.tsx

```diff
@@ -5,9 +5,6 @@ import { useState } from "react";
 // Components
 import MaterialIcon from "../Icon";
 
-// Styles
-import "@suankularb-components/css/dist/css/suankularb-components.min.css";
```

Comment:
I'm not really sure but why is this removed?

Reply:
It still shows up in the build and I can still use the library with `npm link` so I don’t see why we need to import the styles in every single file. We still have to publish it though to see if it’d fully work but I don’t see why not.